composer.lock

```diff
@@ -4604,16 +4604,16 @@
         },
         {
             "name": "j0k3r/graby-site-config",
-            "version": "1.0.198",
+            "version": "1.0.200",
```

Comment:
why this is part of this PR?

Reply:
Because @j0k3r released a new version and we update only the master branch https://github.com/wallabag/wallabag/pull/8235

Reply:
so as discussed should be a dedicated PR, or at least a dedicated commit, and it needs its dedicated entry in the changelog

Reply:
Done through #8250